src/modules/esl-carousel/core/esl-carousel.less

```diff
@@ -0,0 +1,58 @@
+// CORE styles
+esl-carousel {
+  position: relative;
+  display: block;
+  max-width: 100%;
+  // Important to ignore padding/margin hack
+  box-sizing: content-box;
+  margin: -5px;
+  padding: 5px;
+
+  overflow: hidden; // Fallback
+  /* stylelint-disable-next-line */
+  overflow: clip;
+
+  [data-slides-area] {
+    position: relative;
+    width: 100%;
+    display: flex;
+    gap: 20px;
+    z-index: 0;
+    transform: translate3d(0px, 0px, 0px);
+  }
+
+  &.esl-carousel-vertical [data-slides-area] {
+    flex-direction: column;
+  }
+  &.esl-carousel-horizontal [data-slides-area] {
+    flex-direction: row;
+  }
+
+  &[dragging] {
+    cursor: grabbing;
+  }
+
+  &[dragging] [data-slides-area],
+  &[animating] [data-slides-area] {
+    user-select: none;
+    touch-action: pan-y;
+    pointer-events: none;
+  }
+}
+
+esl-carousel-slide {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  flex: 0 0 auto;
+
+  // TODO: Do we need to set display: block?
+  display: flex;
```

Comment:
Is the question still relevant?

Reply:
Yea, I think so.
It is unexpected for a user to see flexbox by default inside the content area.
But It might be better to deal with it in a separate PR.

Reply:
Doublecheck examples without flexbox on slides, it appears to be a small non-breaking change. So fixed)